### PR TITLE
Add tooltip when drawing a polygon that shows how to end drawing

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: src/components/mapgl/mapgl.component.ts [hunk 1]
```diff
@@ -495,9 +504,18 @@ export class MapglComponent implements OnInit, AfterViewInit, OnChanges {
...
+    this.finishDrawTooltip = document.getElementById('polygon-finish-draw-tooltip');
+    this.validateDrawTooltip = document.getElementById('polygon-validate-draw-tooltip');
+    fromEvent<MouseEvent>(window, 'mousemove').subscribe((e) => {
+      const x = e.clientX;
+      const y = e.clientY;
+      this.finishDrawTooltip.style.top = (y + 20) + 'px';
+      this.finishDrawTooltip.style.left = (x + 20) + 'px';
+      this.validateDrawTooltip.style.top = (y + 20) + 'px';
+      this.validateDrawTooltip.style.left = (x + 20) + 'px';
+    });
```
Comment (sebbousquet) [inline src/components/mapgl/mapgl.component.ts]: This event mustn't be listen permanently, only when the user draw a polygon or update an existing one

File: src/components/mapgl/model/ValidGeomDrawPolygonMode.ts [hunk 2]
```diff
@@ -13,7 +14,24 @@ ValidGeomDrawPolygonMode.fireInvalidGeom = function (feature) {
...
+ValidGeomDrawPolygonMode.fireOnClick = function () {
+  this.map.fire('draw.onClick', 'point drawn');
+};
+
+ValidGeomDrawPolygonMode.fireOnStop = function () {
+  this.map.fire('draw.onStop', 'draw end');
+};
+
+ValidGeomDrawPolygonMode.onTap = ValidGeomDrawPolygonMode.onClick = function(state, e) {
+  this.fireOnClick();
```
Comment (sebbousquet) [inline src/components/mapgl/model/ValidGeomDrawPolygonMode.ts]: `fireOnClick` is call if a click is done anywhere, should not it be called only if isn't a vertex ?

File: src/components/mapgl/mapgl.component.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/components/mapgl/mapgl.component.ts]: fixed

File: src/components/mapgl/model/ValidGeomDrawPolygonMode.ts (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/components/mapgl/model/ValidGeomDrawPolygonMode.ts]: fixed

**Pull request merged**